tests/neuro_san/client/assert_forwarder.py

```diff
@@ -0,0 +1,145 @@
+
+        :param expr: Expression to test
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIs(self, first: Any, second: Any, msg: str = None):
+        """
+        Assert that the first and second are the same object
+
+        :param first: First comparison element
+        :param second: Second comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIsNot(self, first: Any, second: Any, msg: str = None):
+        """
+        Assert that the first and second are not the same object
+
+        :param first: First comparison element
+        :param second: Second comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIsNone(self, expr: Any, msg: str = None):
+        """
+        Assert that the expression is None
+
+        :param expr: Expression to test
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIsNotNone(self, expr: Any, msg: str = None):
+        """
+        Assert that the expression is not None
+
+        :param expr: Expression to test
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIn(self, member: Any, container: Any, msg: str = None):
+        """
+        Assert that the member is in the container
+
+        :param member: Member comparison element
+        :param container: Container comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertNotIn(self, member: Any, container: Any, msg: str = None):
+        """
+        Assert that the member is not in the container
+
+        :param member: Member comparison element
+        :param container: Container comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertIsInstance(self, obj: Any, cls: Any, msg: str = None):
+        """
+        Assert that the obj is an instance of the cls
+
+        :param obj: object instance comparison element
+        :param cls: Class comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
+
+    # pylint: disable=invalid-name
+    def assertNotIsInstance(self, obj: Any, cls: Any, msg: str = None):
+        """
+        Assert that the obj is not an instance of the cls
+
+        :param obj: object instance comparison element
+        :param cls: Class comparison element
+        :param msg: optional string message
+        """
+        raise NotImplementedError
```

Comment:
It's possible we might want to add more later on to more or less match what unittest.TestCase gives us today.